TelCo.ColorCoder/Colors.cs

```diff
@@ -0,0 +1,51 @@
+﻿using System;
+using TelCo.ColorCoder;
+
+namespace TelCo.ColorCoder
+{
+   class Colors
```

Comment:
again naming: colors would have list of colors - the class name doesnt indicate the functionality of the mapping of colr to pair number and pair number to color that the class provides